packages/browser/src/login/oidc/incomingRedirectHandler/AuthCodeRedirectHandler.ts

```diff
@@ -105,6 +105,14 @@ export class AuthCodeRedirectHandler implements IIncomingRedirectHandler {
       this.issuerConfigFetcher
     );
 
+    const iss = url.searchParams.get("iss") as string;
+
+    if (iss && iss !== issuerConfig.issuer) {
```

Comment:
```suggestion
    if (typeof iss === "string" && iss !== issuerConfig.issuer) {
```
This complements my previous comment about casting `iss` to `string`, and just makes the test more explicit.